src/pages/trade/ui/route-book.tsx

```diff
@@ -1,60 +1,212 @@
+          <RouteDisplay tokens={trace.hops.map(valueView => getSymbolFromValueView(valueView))} />
+        </td>
+      ) : (
+        <>
+          <td
+            className={
+              isSell ? 'text-[#F17878] text-xs relative' : 'text-[#55D383] text-xs relative'
+            }
+          >
+            {trace.price}
+          </td>
+          <td className='relative text-xs text-right text-white'>{trace.amount}</td>
+          <td className='relative text-xs text-right text-white'>{trace.total}</td>
+          <td className='relative text-xs text-right'>
+            <HopCount count={trace.hops.length} />
+          </td>
+        </>
+      )}
+    </tr>
+  );
+};
+
+export const ROUTEBOOK_TABS = [{ label: 'Route book', value: 'routes' }];
+
 const RouteBookData = observer(({ bookData: { multiHops } }: { bookData: RouteBookResponse }) => {
+  const pair = usePathSymbols();
+
+  const sellRelativeSizes = calculateRelativeSizes(multiHops.sell);
+  const buyRelativeSizes = calculateRelativeSizes(multiHops.buy);
+
   return (
-    <div className='h-[512px] text-white'>
-      <table className='w-full'>
-        <thead>
-          <tr>
-            <th>Price</th>
-            <th className='text-right'>Amount</th>
-            <th className='text-right'>Total</th>
-            <th className='text-right'>Hops</th>
-          </tr>
-        </thead>
-        <tbody>
-          {multiHops.sell.map(trace => (
-            <tr key={trace.price + trace.total} style={{ color: 'red' }}>
-              <td className='text-left tabular-nums'>{trace.price}</td>
-              <td className='text-right tabular-nums'>{trace.amount}</td>
-              <td className='text-right tabular-nums'>{trace.total}</td>
-              <td className='text-right tabular-nums'>{trace.hops.length}</td>
-            </tr>
-          ))}
-          {multiHops.buy.map(trace => (
-            <tr key={trace.price + trace.total} style={{ color: 'green' }}>
-              <td className='text-left tabular-nums'>{trace.price}</td>
-              <td className='text-right tabular-nums'>{trace.amount}</td>
-              <td className='text-right tabular-nums'>{trace.total}</td>
-              <td className='text-right tabular-nums'>{trace.hops.length}</td>
+    <div className='flex flex-col max-w-full border-y border-[#262626]'>
+      <div className='flex items-center gap-2 px-4 h-11 border-b border-[#262626]'>
+        <Tabs
+          value={'routes'}
+          onChange={() => {
+            undefined;
+          }}
+          options={ROUTEBOOK_TABS}
+          actionType='accent'
+        />
+      </div>
+
+      <div className='flex-1'>
+        <table className='w-full'>
+          <thead>
+            <tr className='text-xs text-gray-400'>
+              <th className='py-[8px] font-normal text-left'>Price({pair.quoteSymbol})</th>
+              <th className='py-[8px] font-normal text-right'>Amount({pair.baseSymbol})</th>
```

Comment:
suggestion: Can we convert the px amount to a default tailwind choice if possible? I believe py-[8px] is the same as py-2 right?